Packs/McAfee_Advanced_Threat_Defense/Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense_test.py

```diff
@@ -14,6 +14,56 @@ def set_params(mocker):
     mocker.patch.object(demisto, 'params', return_value=integration_params)
 
 
+@pytest.mark.parametrize('args', [
+    ({'submitType': '2', 'entryID': 'entry_id'}),
+    ({'submitType': '2', 'url': 'url'}),
+    ({'submitType': '0', 'entry_id': 'entry_id', 'url': 'url'}),
+    ({'submitType': '1', 'entryID': 'entry_id', 'url': 'url'}),
+    ({'submitType': '3', 'entryID': 'entry_id', 'url': 'url'}),
+    ({'submitType': '1', 'entryID': 'entry_id'}),
+    ({'submitType': '3', 'entryID': 'entry_id'}),
+    ({'submitType': '0', 'url': 'url'}),
+    ({'submitType': '4'}),
+])
+def test_handling_errors_with_file_upload_command(args):
+    """
+    Given:
+        submitType , submitType , url arguments
+
+    When:
+        Execute command atd-file-upload
+
+    Then:
+        Arguments' validation - returns error if one of the given
+        arguments does not fit the command's structure
+    """
+    from McAfee_Advanced_Threat_Defense import handling_errors_with_file_upload_command
+    with pytest.raises(SystemExit) as e:
```

Comment:
you should check that it raises the appropriate error using `match`. e.g:
    with pytest.raises(Exception, match="When submitType is 2 You must submit both url and entryID"):
        handling_errors_with_file_upload_command(args={'submitType': '2', 'url': 'url'})

Reply:
Done, did something different